src/sidebar/components/share-dialog.js

```diff
@@ -8,15 +8,15 @@ function ShareDialogController($scope, $element, analytics, store) {
 
   function updateViaLink(frames) {
     if (!frames.length) {
-      self.viaPageLink = '';
+      self.sharePageLink = '';
       return;
     }
 
     // Check to see if we are on a via page. If so, we just return the URI.
     if (frames[0].uri.indexOf(VIA_PREFIX) === 0) {
-      self.viaPageLink = frames[0].uri;
+      self.sharePageLink = frames[0].uri;
     } else {
-      self.viaPageLink = VIA_PREFIX + frames[0].uri;
+      self.sharePageLink = 'https://hyp.is/go?url='+frames[0].uri;
```

Comment:
@robertknight : I don't know if this should be hardcoded this way or if we should derive the `hyp.is` bit from somewhere.

Reply:
Hardcoding this is an OK first step but the sharing URL prefix/template ought to come from the same place that generates `hyp.is` links for annotations. This is ultimately the `BOUNCER_URL` env var in the `h` service.

I would suggest you could approach this in two steps:

1. Hard-code the production URL template as you are doing here.
2. Add the share URL template to the output of `/api/links` from the "h" service and then use the `serviceUrl` service in the client to generate a URL given the links data from the service and the frame URL provided by the client.